### PR TITLE
Add the Snapshot Save and Restore topic

### PR DESCRIPTION
**Pull request by ghost**
Trello card: https://trello.com/c/SBEktYWi/533-document-support-new-backup-restore-command-etcdctl-snapshot-save-restore

### CODE REVIEW

**Comment by tmorriso-rh** [comment]
@anpingli @jiajliu Please review. Thanks.

**Comment by tmorriso-rh** [comment]
[rev_history]
|xref:../admin_guide/backup_restore.adoc#admin-guide-backup-and-restore[Backup and Restore]
|Added the admin_guide/backup_restore.adoc#backup-restore-etcd-snapshot[Snapshot Save and Restore] section.
%

**Comment by tmorriso-rh** [comment]
@openshift/team-documentation PTAL. Rev history in the previous comment.

File: admin_guide/backup_restore.adoc [hunk 1]
```diff
@@ -732,6 +732,39 @@ On each master:
 
 The procedure to add an etcd member is complete.
 
+[[backup-restore-etcd-snapshot]]
+== Snapshot Save and Restore
+
+Snapshot provides commands to restore a snapshot of a running etcd server into
+a fresh cluster. 'snapshot save' writes a point-in-time snapshot of the etcd
```
Comment (mburke5678) [inline admin_guide/backup_restore.adoc]: Should the first sentence include the save part? _Snapshot provides commands to **save and** restore a snapshot
Should the `snapshot save` sentence be in its own paragraph like restore? 
Also, should the sentence start _Snapshot save_ (not the command), like restore?

**Comment by mburke5678** [comment]
@tmorriso-rh A couple of nits. Otherwise LGTM.

File: admin_guide/backup_restore.adoc (hunk 1 above)
Comment (tmorriso-rh) [reply admin_guide/backup_restore.adoc]: Fixed.

**Comment by tmorriso-rh** [comment]
@anpingli @jiajliu Bump. Please review. Thanks.

**Comment by tmorriso-rh** [comment]
@jiajliu I changed the note to read:

[NOTE]
====
`etcdctl snapshot` can only back up v3 API data. For etcdctl to speak to etcd using the v3 API, the API version must be set to version 3 via the `ETCDCTL_API` environment variable. 
====

Does this look OK? Thanks.

**Comment by tmorriso-rh** [comment]
@jiajliu Bump. Please review the latest changes in the files changed and also the comment on the Trello card. Thanks.

**Comment by tmorriso-rh** [comment]
@anpingli @jiajliu  Bump. Please review the latest changes in the files changed and also the comment on the Trello card. Thanks.

**Comment by tmorriso-rh** [comment]
@jiajliu Does this feature also apply to the dedicated branches?

**Comment by tmorriso-rh** [comment]
Contacted @spurtell for review/input.

**Comment by vikram-redhat** [comment]
@tmorriso-rh - this section has been added under enterprise + origin (check the ifdefs). This means that it won't be for dedicated or online docs. We don't need to wait for @spurtell 's ack on this.

**Comment by spurtell** [comment]
@tmorriso-rh @vikram-redhat this topic should not be included for online/dedicated.

**Comment by tmorriso-rh** [comment]
Thanks @spurtell. 

@vikram-redhat Please see Jia's latest comment on the Trello card. It seems that QE would like to add more info about the complete etcd cluster restore process. I will wait to merge this PR, which means it may not make the release tomorrow. Thoughts? Thanks.

**Comment by tmorriso-rh** [comment]
Bump. @jiajliu Please review the latest comment on the Trello card.

**Comment by vikram-redhat** [comment]
Closing this as per advice from @sdodson as the current documentation is sufficient (see notes in trello board).

**Pull request closed without merging**